common4j/build.gradle

```diff
@@ -33,6 +33,7 @@ repositories {
 }
 
 apply from: './versioning/version_tasks.gradle'
+apply from: '../../gradle/versions.gradle'
```

Comment:
This seems interesting, because I think it was working without it earlier

Reply:
looks like you removed it already

Reply:
Yes just removed it. I just figured that it was resolving a small bug I had on dev environment.